isis/tests/Fixtures.cpp

```diff
@@ -31,6 +32,25 @@ namespace Isis {
       }
       testCube->write(line);
     }
+
+    // Add a BandBin group to the cube label
+    Pvl *label = testCube->label();
+    PvlObject& cubeLabel = label->findObject("IsisCube");
+    PvlGroup bandBin("BandBin");
+    PvlKeyword originalBand("OriginalBand", "1");
+    originalBand += "2";
+    originalBand += "3";
+    originalBand += "4";
+    originalBand += "5";
+    originalBand += "6";
+    originalBand += "7";
+    originalBand += "8";
+    originalBand += "9";
+    originalBand += "10";
+    bandBin += originalBand;
+    cubeLabel.addGroup(bandBin);
+    testCube->close();
+    testCube->open(path);
```

Comment:
This needs to be re-opened as read/write. It's causing an error when other tests try adding stuff to its label.

Reply:
Thanks for catching this!